rplugin/python3/deoplete/sources/beancount.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, vim):
         self.name = 'beancount'
         self.mark = '[bc]'
         self.filetypes = ['beancount']
+        self.rank = 500
```

Comment:
What does this do?  I've disabled all the other matching types for my beancount files because they mostly mess things up for me.